src/modules/commands/index.ts

```diff
@@ -74,6 +78,14 @@ export default class CommandsModule<T extends VoltareClient<any>> extends Voltar
     return command;
   }
 
+  /**
+   * Registers multiple commands from one file.
+   * @param commands The commands to register
+   */
+  private registerMultipleCommands(commands: any) {
```

Comment:
This is *kinda* unnecessary if its just iterating an array. You can move this up to `#register()`